packages/blade/src/components/Charts/BarChart/BarChart.web.tsx

```diff
@@ -25,6 +26,24 @@ import { getComponentId } from '~utils/isValidAllowedChildren';
 import { makeAnalyticsAttribute } from '~utils/makeAnalyticsAttribute';
 import type { DataAnalyticsAttribute, TestID } from '~utils/types';
 
+const MotionRectangle = (props: MotionRectProps): React.ReactElement => {
+  const { fillOpacity, barIndex, initialOpacity, ...restProps } = props;
+  return (
+    <LazyMotion features={domAnimation}>
```

Comment:
LazyMotion is added by consumers in their applications. We shouldn't add it in components